### PR TITLE
Fix glyph 2

### PR DESCRIPTION
**Pull request by langjoh**

### CODE REVIEW

**Comment by langjoh** [comment]
An extra definition of the glyphicon class (which was in a stale scss file) restricted every glyphicon to 12px, which is too small. We erased the whole block.
Further we added the font for the Hafling glyphicons in the woff2 format, as the browser gave an error message that it was missing.

File: app/assets/stylesheets/custom/_drops.scss [hunk 1]
```diff
@@ -26,8 +26,5 @@
   border-radius: 20px;
 }
 
-.glyphicon {
-  width: 12px;
-  color: #f50;
-}
+
```
Comment (rstammer) [inline app/assets/stylesheets/custom/_drops.scss]: I'd vote for another tiny whitespace cleanup and drop the unused white lines at the end 
gif-fantasia-broom2

**Comment by rstammer** [comment]
👍  Good job!

**Comment by berlintam** [comment]
awesome

**Pull request merged**